src/clj_pdf/core.clj

```diff
@@ -654,6 +654,14 @@
   [_ width height]
   (new Rectangle width height))
 
+(defn- split-classes-from-tag
+  [tag]
+  (map keyword (split (name tag) #"\.")))
```

Comment:
@yogthos normalizing the tag to a keyword here, so this should be okay for deserialized json when we check `(= tag :pdf-cell)`